spec/acceptance/standard_spec.rb

```diff
@@ -6,12 +6,12 @@
     # Using puppet_apply as a helper
     it 'should work with no errors based on the example' do
       pp = <<-EOS
-        class { 'hashi_stack::repo': }
+        class { 'hashi_stack::repo': } -> package { 'packer': ensure => installed }
```

Comment:
You can use multiple lines. How about:
```suggestion
        include hashi_stack::repo
        package { 'packer':
          ensure  => installed,
          require => Class['Hashi_stack::Repo'],
        }
```